hdfs_query.c

```diff
@@ -19,25 +19,73 @@
 #include "hdfs_fdw.h"
 
 double
+hdfs_find_row_count(char *src)
```

Comment:
Please explain the logic in well defined comments, I am unable to understand the code.